release_management/create_release.py

```diff
@@ -68,12 +68,15 @@ class RustCrates:
     """
 
     @staticmethod
-    def set_up_subcommand(subcmd, subparsers, target_func):
+    def set_up_subcommand(
+        subcmd, subparsers, target_func, add_github_release_option=False
```

Comment:
Make ```add_github_release_option``` a _mandatory_ keyword option, so

```
def set_up_subcommand(
        subcmd, subparsers, target_func, *, add_github_release_option=False
)
```

Note the '*'

Reply:
Resolved.